spec/components/schemas/content/bases/buttons-list.ts

```diff
@@ -6,8 +6,9 @@ export const buttonsList: SchemaObject = {
   type: 'array',
   title: 'Buttons',
   description: `Vertical list of buttons displayed inside a card. These will *not* fade away once one a button is clicked.<br>
-    <br>*[RCS](#tag/RCS): A maximum of 4 buttons is supported, but they might not be shown if they do not fit in the available space.*
-    <br>*[Facebook](#tag/Facebook): A maximum of 3 buttons is supported. Only \`text\` and \`link\` buttons are supported.*`,
+    <br>*[RCS](#tag/RCS): A maximum of 4 buttons are supported, but they might not be shown if they do not fit in the available space.*
+    <br>*[Google Business Messages](#tag/Google-Business-Messages): A maximum of 4 buttons are supported. Types supported: \`dial\`, \`link\`, \`text\`.*
```

Comment:
Será que dá pra manter na mesma ordem que os tipos são declarados no enum, e com um and? `text, link and dial` (a última virgula sempre vira "e" em textos).

E inverte pra `Supported types` pra ficar correto em inglês.

Reply:
Feito. Muito obrigado!